compiler/qsc_frontend/src/typeck/rules.rs

```diff
@@ -582,6 +588,13 @@ impl<'a> Context<'a> {
                         .expect("local should have type")
                         .clone(),
                 ),
+                Some(Res::ExportedItem(item, _)) => {
+                    // get the underlying item this refers to
+                    let item_scheme = self.globals.get(item).expect("item should have scheme");
```

Comment:
I'm a bit unfamiliar with the term "item scheme". I couldn't find a reference to it in the functions used in this part of the logic. What do you mean by it?

Reply:
Thanks for the explanation!

Reply:
I think this framework is a bit out-of-date. It comes from when all namespace items could be used as a callable, including UDT's (referring to their constructor function). Moving forward, I don't think items other than callables should have schemes (and UDT's until we remove them).

Reply:
The scheme describes the callable type associated with the item.